graph/config/cytoscape/cytoscape.go

```diff
@@ -59,26 +59,27 @@ type NodeData struct {
-	Version         string              `json:"version,omitempty"`
-	Service         string              `json:"service,omitempty"`         // requested service for NodeTypeService
-	Aggregate       string              `json:"aggregate,omitempty"`       // set like "<aggregate>=<aggregateVal>"
-	DestServices    []graph.ServiceName `json:"destServices,omitempty"`    // requested services for [dest] node
-	Traffic         []ProtocolTraffic   `json:"traffic,omitempty"`         // traffic rates for all detected protocols
-	HasCB           bool                `json:"hasCB,omitempty"`           // true (has circuit breaker) | false
-	HasMissingSC    bool                `json:"hasMissingSC,omitempty"`    // true (has missing sidecar) | false
-	HasVS           bool                `json:"hasVS,omitempty"`           // true (has route rule) | false
-	IsDead          bool                `json:"isDead,omitempty"`          // true (has no pods) | false
-	IsGroup         string              `json:"isGroup,omitempty"`         // set to the grouping type, current values: [ 'app', 'version' ]
-	IsInaccessible  bool                `json:"isInaccessible,omitempty"`  // true if the node exists in an inaccessible namespace
-	IsMisconfigured string              `json:"isMisconfigured,omitempty"` // set to misconfiguration list, current values: [ 'labels' ]
-	IsOutside       bool                `json:"isOutside,omitempty"`       // true | false
-	IsRoot          bool                `json:"isRoot,omitempty"`          // true | false
-	IsServiceEntry  string              `json:"isServiceEntry,omitempty"`  // set to the location, current values: [ 'MESH_EXTERNAL', 'MESH_INTERNAL' ]
-	IsUnused        bool                `json:"isUnused,omitempty"`        // true | false
+	NodeType         string              `json:"nodeType"`
+	Namespace        string              `json:"namespace"`
+	Workload         string              `json:"workload,omitempty"`
+	App              string              `json:"app,omitempty"`
+	Version          string              `json:"version,omitempty"`
+	Service          string              `json:"service,omitempty"`          // requested service for NodeTypeService
+	Aggregate        string              `json:"aggregate,omitempty"`        // set like "<aggregate>=<aggregateVal>"
+	DestServices     []graph.ServiceName `json:"destServices,omitempty"`     // requested services for [dest] node
+	Traffic          []ProtocolTraffic   `json:"traffic,omitempty"`          // traffic rates for all detected protocols
+	HasCB            bool                `json:"hasCB,omitempty"`            // true (has circuit breaker) | false
+	HasMissingSC     bool                `json:"hasMissingSC,omitempty"`     // true (has missing sidecar) | false
+	HasVS            bool                `json:"hasVS,omitempty"`            // true (has route rule) | false
+	HealthAnnotation []interface{}       `json:"healthAnnotation,omitempty"` // requested healthAnnotation for NodeType
```

Comment:
I guess "Annotation" shouldn't belong to this domain, I guess it's better to name it "HealthConfig" and I think type should be a `string` if I'm not wrong and I missed something.